src/db/ActivePIN.db.ts

```diff
@@ -126,11 +129,50 @@ export async function deletePin(
                     alteredByUsername: expiredByUsername,
                 },
             );
+
+            try {
+                if (PINToDelete) {
+                    const personalisation = {
+                        property_address: requestBody.propertyAddress,
+                        pin: PINToDelete.pin,
+                    };
+
+                    const emailTemplateId: string =
+                        process.env.GC_NOTIFY_EXPIRE_EMAIL_TEMPLATE_ID!;
+                    const phoneTemplateId: string =
+                        process.env.GC_NOTIFY_EXPIRE_PHONE_TEMPLATE_ID!;
+
+                    if (requestBody.phoneNumber) {
+                        // eslint-disable-next-line @typescript-eslint/no-unused-vars
+                        const gcNotifyResponse =
```

Comment:
Another problem. What happens if we include both a phone number and an email. One goes through but not the other. Is this still an error?

Reply:
And moreover, should all of this be in a transaction? Let's say we delete but the GCNotify doesn't go through. Shouldn't the deletion be reversed?